docs/chapters/02_getting-started.md

```diff
@@ -600,25 +601,31 @@ You can also clean the compiled code by running:
 boost clean
 ```
 
-Now, let's deploy our application to the cloud to see it working. It is as simple as running
+Now, let's run our application to see it working. It is as simple as running:
+
+```bash
+boost start -e local
+```
+
+Or, we can deploy our application to the cloud with no additional changes by running
 the deploy command:
 
 ```bash
 boost deploy -e production
 ```
 
-And here it comes the Booster magic! ✨ When running the deploy command, Booster will handle the creation of all the resources, *like Lambdas, API Gateway,* and the "glue" between them; *permissions, events, triggers, etc.* It even creates a fully functional GraphQL API!
+This is the Booster magic! ✨ When running the start or the deploy commands, Booster will handle the creation of all the resources, _like Lambdas, API Gateway,_ and the "glue" between them; _permissions, events, triggers, etc._ It even creates a fully functional GraphQL API!
 
 > [!NOTE] Deploy command automatically builds the project for you before performing updates in the cloud provider,
```

Comment:
```suggestion
> [!NOTE] The `deploy` command automatically builds the project for you before performing updates in the cloud provider,
```
I'm not sure if that "deploy" will be transformed with those commas since it's already inside the ">" block, but I'd love to add the "the" before 🙏